tests/conftest.py

```diff
@@ -0,0 +1,175 @@
+import json
+from typing import Tuple
+
+import pytest
+import requests
+from _pytest.fixtures import FixtureRequest
```

Comment:
🤷🏻‍♂️ it's definitely nice having the type hints/autocomplete in the editor. I'll read into some standards/suggestions and will post back here what I find. I'm not sure _how bad_ it is importing the private package.

Reply:
So I had just taken the recommended import path from vs code, which was incorrect of me to do since `FixtureRequest` is actually available from the public `pytest` package 🤦🏻‍♂️  https://github.com/pytest-dev/pytest/blob/ee93557ef33c03d7a792af1921806b54fe7c7a89/src/pytest/__init__.py#L99

Fixed in https://github.com/mozilla-services/kinto-dist/pull/1953/commits/df2c1d2b2afc3f162c63e4aa3cb0a8c223003f37